docs/submission/submitted-data.md

```diff
@@ -8,107 +8,90 @@ Program members can track the progress of their program’s clinical and molecul
-![Dashbard Donor Release Summary](/assets/submission/dashboard-donor-release-card.png)
+_Coming Soon:_ The status bar in this card shows how many program donors have fully released files (dark blue bar), partially released files (light blue bar) and no released files (white bar) out of the total number of donors committed to ARGO by the program.  
 
-## Program Workspace Status
+### Program Workspace Status
 
 The **Program Workspace** consists of the sample registration as well as the clinical submission areas.
-![Dashbard Program Workspace Status](/assets/submission/dashboard-program-workspace-card.png)
 
 > All program members share the same workspace, so members are encouraged to communicate with each other when a sample registration or clinical submission is in progress.
 
 **Sample Registration Progress:** this progress bar will show if there are uploaded samples that haven’t been registered yet. A blank status bar means this area is free to register samples.
 
 **Clinical Submission Progress:** this progress bar will show if a clinical submission currently has uploaded files or if it has been validated and is ready for sign off. If a submission is signed off without any updates to previously submitted data, this progress bar will become blank again, signifying that this area is free to submit clinical data. If the progress bar is in “pending approval” state, this means the clinical submission area has been locked and is under review by the DCC. Once the DCC approves the clinical data updates, the progress bar will become blank and the clinical submission area is free for a new submission.
 
-## Completed Core Clinical Data
+### Clinical Completeness Data Summary
 
-_Coming Soon:_ This line graph shows the number of donors that are clinically complete over time.
+This line graph compares how many donors have complete clinical data over time. 
+- Check out [Clinical Data Validation Rules](/docs/submission/clinical-data-validation-rules) for more information on how clinical completion is calculated.
+- As soon as a donors clinical data is completed, this chart will update in real-time. 
 
-## Molecular Data Summary
+### Molecular Data Summary
 
-_Coming Soon:_ This line graph compares how many donors have been processed through the [DNA-Seq Pipeline](/docs/analysis-workflows/dna-pipeline): Raw Read Submission, Alignment and Sanger VC.
+This line graph compares how many donors have been processed through the [DNA-Seq Pipeline](/docs/analysis-workflows/dna-pipeline) (Raw Read Submission, Alignment, Sanger VC...) over time.
+- Use the legend to toggle different pipeline steps on/off to check progress of different processing steps. 
+- This chart is kept up-to-date as data is processed in real-time.  While the data may be processed on the chart, it may not yet be released to the program if a donors is lacking complete clinical data. 
```

Comment:
reword suggestion
`This chart is updated in real-time. While the data may show that a donor has been processed in the molecular summary chart, the donor files may not yet be released to the program if that donor is lacking complete clinical data. `